public/base-locales/pl.json

```diff
@@ -85,40 +85,40 @@
   "locale_selection_pt-br": "Português",
   "locale_selection_sv": "Svenska",
   "locale_selection_th": "ไทย",
-  "import": "Importuj",
-  "export": "Eksportuj",
-  "logout": "Wyloguj",
+  "import": "Import ustawień",
+  "export": "Eksport ustawień",
+  "logout": "Wyloguj się",
   "local_storage_cleared": "Lokalna pamięć została wyczyszczona!",
   "iv_or_filter": "IV OR Filtr",
   "polygons": "Obszary",
   "raids": "Raidy",
-  "ex_eligible": "EX Gymy",
+  "ex_eligible": "Gymy EXowe",
   "in_battle": "W trakcie walki",
-  "all_pokestops": "Pokestopy",
-  "all_gyms": "Gymy",
+  "all_pokestops": "Wszystkie Pokestopy",
+  "all_gyms": "Wszystkie Gymy",
   "lures": "Lury",
-  "quests": "Zadania",
+  "quests": "Questy",
```

Comment:
After a change, make sure you replace all words. Not like in this case where you replaced only part of strings to `Questy`.